cmd/demo/main.go

```diff
@@ -15,6 +15,8 @@ func main() {
 	defer w.Destroy()
 	w.SetTitle("Minimal webview example")
 	w.SetSize(800, 600, webview2.HintFixed)
-	w.Navigate("https://en.m.wikipedia.org/wiki/Main_Page")
+	//w.Navigate("https://en.m.wikipedia.org/wiki/Main_Page")
```

Comment:
Any chance we can change this back to the original please? Only because I'm likely to send these changes upstream. Thanks!